subprojects/snapshots/src/main/java/org/gradle/internal/fingerprint/impl/DefaultCurrentFileCollectionFingerprint.java

```diff
@@ -118,4 +120,14 @@ public void accept(FileSystemSnapshotVisitor visitor) {
             root.accept(visitor);
         }
     }
+
+    @Override
+    public String toString() {
+        Map<Boolean, List<Map.Entry<String, FileSystemLocationFingerprint>>> collect = fingerprints.entrySet()
+            .stream()
+            .collect(Collectors.partitioningBy(entry -> entry.getValue() instanceof IgnoredPathFileSystemLocationFingerprint));
+        List<Map.Entry<String, FileSystemLocationFingerprint>> includedFingerprints = collect.get(false);
+        List<Map.Entry<String, FileSystemLocationFingerprint>> ignoredFingerprints = collect.get(true);
+        return identifier + "{fingerprints=" + includedFingerprints + ", ignoredFingerprints=" + ignoredFingerprints + "}";
```

Comment:
When the fingerprint is an `IgnoredPathFileSystemLocationFingerprint`, then that doesn't mean that the fingerprint is ignored - it means that the path of the fingerprint is ignored.
The `toString` method already handles that well, no need to print those fingerprints separately.